### PR TITLE
usage example for sprite set position

### PR DESCRIPTION
**Pull request by kay-kaushik**
# Description
This pull request features usage exaples for sprite_set_position(). The usage example is clear and concise to help the user understand how to implement the function.

# Features
+ clear and concise code
+ gifs
+ images
+ perfect example

# Changes
5 files have been added. 2 code, 1 text and 1 zip file and 1 gif.

# Benefits
These high quality usage examples give more context to the functions. This will help users reading the documentation to better understand the function. This will result in faster development.

# Testing
All the changes have been tested on Kartik Kaushik's system

# Testing Images

image

### CODE REVIEW

**Review by omckeon — CHANGES_REQUESTED** [review]
Just need to update a few things mentioned in the comments

**Comment by omckeon** [comment]
Make sure you have run `npm run build` successfully as well. (The check should not be failing.)

File: src/assets/usage-examples-code/sprites/sprite_set_position/sprite_set_position-9-sprite-s-pos.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### sprite_set_position() usage example
```
[the remark this answers is not in the document]
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_position/sprite_set_position-9-sprite-s-pos.txt]: I have made the changes

File: public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png
[the remark this answers is not in the document]
Comment (kay-kaushik) [reply public/usage-examples-images-gifs/sprites/protSpriteSheet-R.png:1]: Done

File: src/assets/usage-examples-code/sprites/sprite_set_position/sprite_set_position-9-sprite-s-pos.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+The following code is an example of using sprite set position which sets the new x and y position of a sprite on a splashkit window.
```
[the remark this answers is not in the document]
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_position/sprite_set_position-9-sprite-s-pos.txt]: I have done as asked.

**Review by NidhishaPahade — COMMENTED** [review]
Everything looks good  to me.

**Pull request merged**